dvc/command/experiments.py

```diff
@@ -482,8 +496,26 @@ def show_experiments(
         row_styles=row_styles,
         csv=csv,
         markdown=markdown,
+        html=html,
+        **extra_render_args,
     )
 
+    if html and kwargs.get("open"):
```

Comment:
I guess it is better to extract this opening of browsers and merge it with 
https://github.com/iterative/dvc/blob/3486435a7fcc76e031f7ed23335a7354441da634/dvc/command/plots.py#L69-L82

Reply:
And @skshetry what do you think should it go with render with an argument or keep it in this show_experiment function?

Reply:
Yeah. @skshetry already suggested that in https://github.com/iterative/dvc/pull/6913#discussion_r748392459 So I guess now it makes even more sense as we have another place where its used

Reply:
I think it should receive a relpath and construct the abspath inside for non WSL cases

Reply:
Opened #6993